packages/log/manifest.yml

```
@@ -4,13 +4,13 @@ title: Custom Logs
 description: >-
   Collect custom logs with Elastic Agent.
 type: input
-version: 2.3.3
+version: 2.3.4
```

Nit. When making changes to the kibana version constraint is a good idea to increase the minor version of the package, in case backports are needed only for the previous set of kibana versions.
```suggestion
version: 2.4.0
```